tsdb/series_file.go

```diff
@@ -74,12 +78,12 @@ func (f *SeriesFile) Open() error {
 	return nil
 }
 
-// Close unmaps the data file.
-func (f *SeriesFile) Close() (err error) {
```

Comment:
Please change this method to `close() (err error) {` and remove the refs lock call.

Reply:
changed as suggested